src/ml/neural_net/weight_init.cpp

```diff
@@ -40,28 +40,51 @@ xavier_weight_initializer::xavier_weight_initializer(
   lstm_weight_initializers lstm_weight_initializers::create_with_xavier_method(
     size_t input_size, size_t state_size, std::mt19937* random_engine)
   {
-  xavier_weight_initializer i2h_init_fn(input_size, state_size, random_engine);
-  xavier_weight_initializer h2h_init_fn(state_size, state_size, random_engine);
+    xavier_weight_initializer i2h_init_fn(input_size, state_size,
+                                          random_engine);
+    xavier_weight_initializer h2h_init_fn(state_size, state_size,
+                                          random_engine);
 
-  lstm_weight_initializers result;
+    lstm_weight_initializers result;
 
-  result.input_gate_weight_fn = i2h_init_fn;
-  result.forget_gate_weight_fn = i2h_init_fn;
-  result.block_input_weight_fn = i2h_init_fn;
-  result.output_gate_weight_fn = i2h_init_fn;
+    result.input_gate_weight_fn = i2h_init_fn;
+    result.forget_gate_weight_fn = i2h_init_fn;
+    result.block_input_weight_fn = i2h_init_fn;
+    result.output_gate_weight_fn = i2h_init_fn;
 
-  result.input_gate_recursion_fn = h2h_init_fn;
-  result.forget_gate_recursion_fn = h2h_init_fn;
-  result.block_input_recursion_fn = h2h_init_fn;
-  result.output_gate_recursion_fn = h2h_init_fn;
+    result.input_gate_recursion_fn = h2h_init_fn;
+    result.forget_gate_recursion_fn = h2h_init_fn;
+    result.block_input_recursion_fn = h2h_init_fn;
+    result.output_gate_recursion_fn = h2h_init_fn;
 
-  result.input_gate_bias_fn = zero_weight_initializer();
-  result.forget_gate_bias_fn = zero_weight_initializer();
-  result.block_input_bias_fn = zero_weight_initializer();
-  result.output_gate_bias_fn = zero_weight_initializer();
+    result.input_gate_bias_fn = zero_weight_initializer();
+    result.forget_gate_bias_fn = zero_weight_initializer();
+    result.block_input_bias_fn = zero_weight_initializer();
+    result.output_gate_bias_fn = zero_weight_initializer();
```

Comment:
Why are we changing the indent level here? Didn't we already address this in the last PR?